collect_app/src/main/java/org/odk/collect/android/projects/ProjectSettingsDialog.kt

```diff
@@ -109,6 +109,7 @@ class ProjectSettingsDialog : DialogFragment() {
             }
             projectView.findViewById<TextView>(R.id.project_name).text = project.name
 
+            projectView.contentDescription = getString(R.string.switch_to_project, project.name)
```

Comment:
This might feel a little strange as we're adding data to the view that's only used in tests. It can also be used by screen readers (which is really how we interact with the app in our tests interestingly). Adding this data feels like a win-win as it helps accessibility tools and lets us write simpler tests.